packages/mobile/src/language/Language.test.tsx

```diff
@@ -35,4 +36,25 @@ describe('Language', () => {
     expect(changeLanguageSpy).toHaveBeenCalledWith('es-419')
     expect(navigate).toHaveBeenCalledWith(Screens.OnboardingEducationScreen)
   })
+
+  it('navigates to welcome screen if skip onboarding education screen flag is turned on in redux store', () => {
```

Comment:
could you also add a test for when `shouldSkipOnboardingEducationScreen` is false? Also, I recommend mocking Math.random to make absolutely sure these tests don't pass by random accident. So in this case, you could mock Math.random to return 0.75, which means if the saved value were ignored and a new `shouldSave...` value were selected instead, the test would fail